docs/public-networks/reference/cli/options.md

```diff
@@ -2009,336 +2009,355 @@ Minimum occupancy ratio for a mined block if the transaction pool is not empty.
 
-:::note
+:::tip
 
-Besu ignores this option in networks using [Clique](../../../private-networks/how-to/configure/consensus/clique.md), [IBFT 2.0](../../../private-networks/how-to/configure/consensus/ibft.md), or [QBFT](../../../private-networks/how-to/configure/consensus/qbft.md) consensus protocols.
+In a [free gas network](../../../private-networks/how-to/configure/free-gas.md), ensure the minimum
+gas price is set to zero for every node.
+Any node with a minimum gas price set higher than zero will silently drop transactions with a zero
+gas price.
+You can query a node's gas configuration using [`eth_gasPrice`](../api/index.md#eth_gasprice).
 
 :::
 
-### `miner-enabled`
+### `min-priority-fee`
 
 <Tabs>
 
 <TabItem value="Syntax" label="Syntax" default>
 
 ```bash
---miner-enabled[=<true|false>]
+--min-priority-fee=<minPriorityFeePerGas>
 ```
 
 </TabItem>
 
 <TabItem value="Example" label="Example">
 
 ```bash
---miner-enabled=true
+--min-gas-price=7
 ```
 
 </TabItem>
 
 <TabItem value="Environment variable" label="Environment variable">
 
 ```bash
-BESU_MINER_ENABLED=true
+BESU_MIN_PRIORITY_FEE=7
 ```
 
 </TabItem>
 
 <TabItem value="Configuration file" label="Configuration file">
 
 ```bash
-miner-enabled=true
+min-priority-fee=7
 ```
 
 </TabItem>
 
 </Tabs>
 
-Enables or disables mining when you start the node. The default is `false`.
+The minimum priority fee per gas (in wei) offered by a transaction to be included in a block.
+The default is `0`.
 
-### `miner-extra-data`
+For a running node, use:
+
+* [`miner_getMinPriorityFee`](../api/index.md#minergetminpriorityfee) to get the value.
+* [`miner_setMinPriorityFee`](../api/index.md#minersetminpriorityfee) to change the value.
+
+### `miner-coinbase`
 
 <Tabs>
 
 <TabItem value="Syntax" label="Syntax" default>
 
 ```bash
---miner-extra-data=<Extra data>
+--miner-coinbase=<Ethereum account address>
 ```
 
 </TabItem>
 
 <TabItem value="Example" label="Example">
 
 ```bash
---miner-extra-data=0x444F4E27542050414E4943202120484F444C2C20484F444C2C20484F444C2021
+--miner-coinbase=fe3b557e8fb62b89f4916b721be55ceb828dbd73
 ```
 
 </TabItem>
 
 <TabItem value="Environment variable" label="Environment variable">
 
 ```bash
-BESU_MINER_EXTRA_DATA=0x444F4E27542050414E4943202120484F444C2C20484F444C2C20484F444C2021
+BESU_MINER_COINBASE=fe3b557e8fb62b89f4916b721be55ceb828dbd73
 ```
 
 </TabItem>
 
 <TabItem value="Configuration file" label="Configuration file">
 
 ```bash
-miner-extra-data="0x444F4E27542050414E4943202120484F444C2C20484F444C2C20484F444C2021"
+miner-coinbase="0xfe3b557e8fb62b89f4916b721be55ceb828dbd73"
 ```
 
 </TabItem>
 
 </Tabs>
 
-A hex string representing the 32 bytes included in the extra data field of a mined block. The default is 0x.
+The account you pay mining rewards to.
+You must specify a valid coinbase when you enable mining using the
+[`--miner-enabled`](#miner-enabled) option or the [`miner_start`](../api/index.md#miner_start)
+JSON-RPC API method.
```

Comment:
it might be worth mentioning that miner-coinbase is used as a "fall-back address" for block rewards in proof-of-stake if the consensus layer does not supply a block reward address.